integration_tests/service.proto

```diff
@@ -57,6 +57,32 @@ message HttpDeleteRequest {
   int32 a = 1;
 }
 
+message HTTPGetWithURLSearchParamsRequest {
```

Comment:
It will be good to have the url example here

Reply:
Ah ok, you mean the URL example from the notes section in the README?

Reply:
just a url example will be ok. no need to move the whole thing here.

Reply:
I understand, just a simple request and response message and a simple RPC implementation.

Reply:
I have modified the tests with the URL example. Please let me know if this good or if I have missed something 🙇🏼